### PR TITLE
sketchy ajax call to get circles to show up

### PR DESCRIPTION
**Pull request by jasonshark**
the file wasn't loading, now green circles show up on the map

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -325,23 +325,34 @@
...
-                    //     alert("Hey! You clicked " + data);
-                    // });
-                })
-                .error(function(err) {
-                    console.log("error: " + err);
-                });
+            var vizUrl = "https://rawgit.com/openoakland/measure-g/master/viz.json";
+
+            $.ajax('./viz.json', {
+                dataType: 'jsonp'
```
Comment (jbaldo) [inline index.html]: This is awesome! Thank you!

This line seems like the key. Not too worried about code sketchiness with this project, but wondering... do you see any way to force this data type on the server side?

**Comment by jasonshark** [comment]
haha well we need to set up a server to have a server side right? We could build something with expressjs or django

**Comment by jbaldo** [comment]
Found out you can wrap the json in a closure to make the jsonp stuff work. I guess it's just looking for a callback function object? I don't fully get it, but seems OK for now.

https://groups.google.com/forum/#!topic/cartodb/13VsukLRjnM

**Comment by jasonshark** [comment]
Cool! Looks like [this commit](3b5b6e4238695063135529ba176cdd02a9d13c64) solved it

**Pull request closed without merging**